src/main/java/step2/object/Calculator.java

```diff
@@ -0,0 +1,117 @@
+package step2.object;
+
+
+import step2.exception.BizException;
+
+public class Calculator {
+
+    final String BLANK = " ";
+
+    enum Operation {
+        plus("+") {
```

Comment:
```suggestion
        PLUS("+") {
```

enum 의 멤버들은 상수라는 의미를 강조하기 위해, 항상 대문자로 작성해주시기 부탁드립니다.
컨벤션입니다.